### PR TITLE
chore: add image preload for Screener in N*

### PR DESCRIPTION
**Pull request by layershifter**
This PR adds `<link preload>` via Webpack plugin to ensure that they will be loaded for Screener in N*.

image

### CODE REVIEW

**Comment by layershifter** [comment]
1

File: scripts/webpack/webpack.config.ts [hunk 1]
```diff
@@ -98,6 +99,10 @@ const webpackConfig: webpack.Configuration = {
         reactVis: require('react-vis/package.json').version,
       },
     }),
+    new ResourcePreloadWebpackPlugin({
```
Comment (ecraig12345) [inline scripts/webpack/webpack.config.ts:102]: Can you add a comment about why this was needed?

Comment (layershifter) [reply scripts/webpack/webpack.config.ts:102]: Sure, I am also thinking to run this only for Screener builds

**Comment by layershifter** [comment]
Closing for housekeeping.

**Pull request closed without merging**